doc: Update AWS IoT integration documentation with minor changes (#432)
TheThingsIndustries/lorawan-stack-docs: doc/content/integrations/cloud-integrations/aws-iot/default/deployment-guide.md

{{< /tabs/tab >}}
{{< /tabs/container >}}

> {{< warning >}} Make sure to select the template related to your deployment only. {{</ warning >}}

Reviewer: Can you select multiple templates? The word "only" confuses me here

- **Principal Account ID** {{< distributions "Enterprise" >}}: AWS Account ID that The Things Stack authenticates with.
- **Thing Type Name**: The unique AWS IoT Core thing type name for this integration.
- **Thing Name Scheme**: The name that is given to AWS IoT things when they are created by the integration.

>   - When using DevEUI, the thing name will be like 1122334455667788.

Reviewer 2: ```suggestion
  - When using DevEUI, the name will appear as the numeric DevEUI, i.e 1122334455667788.
```

>   - When using `DevEUI`, the name will appear as the numeric DevEUI, i.e 1122334455667788.

Reviewer: ```suggestion
  - When using `DevEUI`, the name will appear as the numeric DevEUI, i.e `1122334455667788`.
```

>   - When using DevEUI, the thing name will be like 1122334455667788.
>   - When using DeviceID, the thing name will be a combination of the CloudFormation stack name and the device ID as registered in  {{% tts %}}. It would be like `<stack-name>_<device-id>`.

Reviewer: ```suggestion
  - When using a DevEUI, the thing name will be like `1122334455667788`.
  - When using a DeviceID, the thing name will be a combination of the CloudFormation stack name and the device ID as registered in  {{% tts %}}, i.e. `<stack-name>_<device-id>`.
```

Reviewer 2: It should be "When using 'DevEUI/DeviceID'", because that's the name of the scheme.

>   - When using DeviceID, the thing name will be a combination of the CloudFormation stack name and the device ID as registered in  {{% tts %}}. It would be like `<stack-name>_<device-id>`.

Reviewer 2: ```suggestion
  - When using DeviceID, the thing name will be a combination of the CloudFormation stack name and the device ID as registered in  {{% tts %}}, i.e `<stack-name>_<device-id>`.
```
